frontends/web/src/components/devices/bitbox02/bitbox02.tsx

```diff
@@ -207,9 +209,28 @@ class BitBox02 extends Component<Props, State> {
         });
     }
 
+    private handleDeviceNameInput = (event: Event) => {
+        const target = (event.target as HTMLInputElement);
+        const value: string = target.value;
+        this.setState(prevState => ({
```

Comment:
what does this do over `this.setState({ deviceName: value })`?

Reply:
I suppose in this case we don't really need it. Passing a function instead of an object to setState makes sure the state actually ends up being what you expect it to be, especially if multiple setStates are being called at the same and you don't know when the state actually gets changed. But since nothing else is happening to the state in this case, it's kind of an overkill... :)